Minor cleanups for `sizing-calc.md` (#577)
NVIDIA/NeMo-Agent-Toolkit: docs/source/workflows/sizing-calc.md

The sizing calculator uses the [evaluation](evaluate.md) and [profiling](./profiler.md) systems in the NeMo Agent toolkit.

## Overview

This guide assumes that you have an LLM hosted by an isolated GPU cluster, for which you want to perform the sizing calculations for.

:::{note}
Although it is possible to run the sizing calculator against a publicly hosted LLM, the results may not be accurate due to the variability in the performance of public LLMs.
:::

## Getting Started With Sizing a GPU Cluster

To begin, set the configuration file and output directory. For this example we will start with the simple calculator evaluation configuration file, however in a real-world scenario you would use the configuration file of your own workflow you want to size.
```
export CALC_OUTPUT_DIR=.tmp/sizing_calc/
export CONFIG_FILE=${CALC_OUTPUT_DIR}config-sizing-calc.yml

mkdir -p ${CALC_OUTPUT_DIR}

cp examples/evaluation_and_profiling/simple_calculator_eval/configs/config-sizing-calc.yml $CONFIG_FILE
```

> Edit `.tmp/sizing_calc/config-sizing-calc.yml` file adding a `base_url` parameter for the `llms.nim_llm` section for your cluster, and changing the `llms.nim_llm.model_name` if needed.

Reviewer: Is "if needed" for the entire sentence or for just the `llms.nim_llm.model_name`?

Reviewer: ```suggestion
Edit `.tmp/sizing_calc/config-sizing-calc.yml` file by adding a `base_url` parameter for the `llms.nim_llm` section for your cluster. Then, if needed, change the `llms.nim_llm.model_name`.
```

For a locally hosted NIM this might look like:
```yaml
llms:
  nim_llm:
    _type: nim
    base_url: "http://localhost:8000/v1"
    model_name: meta/llama-3.3-70b-instruct
```

### Step 1: Gather Metrics
Collect performance data at different concurrency levels:
```
aiq sizing calc --config_file $CONFIG_FILE --calc_output_dir $CALC_OUTPUT_DIR --concurrencies 1,2,4,8,16,32 --num_passes 2
```

:::{note}

> Depending on the number of concurrencies, the number of passes, and the size of the cluster being tested this could take several minutes to run.

Reviewer: ```suggestion
Depending on the number of concurrencies, the number of passes, and the size of the cluster being tested, this could take several minutes to run.
```

**Summary Table**

The summary table provides an overview of the per-concurrency metrics.

> - The `P95 LLM Latency` (95th percentile LLM latency) column contains the latency in seconds across all LLM invocations. If multiple models are used, the value will trend towards the latency of the model with the highest latency.

Reviewer: ```suggestion
- The `P95 LLM Latency` (95th percentile LLM latency) column contains the latency, in seconds, across all LLM invocations. If multiple models are used, the value will trend towards the latency of the model with the highest latency.
```

> - The `P95 WF Runtime` (95th percentile workflow runtime) column contains the response time in seconds of the workflow and is computed across all runs at the specified concurrency.

Reviewer: ```suggestion
- The `P95 WF Runtime` (95th percentile workflow runtime) column contains the response time, in seconds, of the workflow and is computed across all runs at the specified concurrency.
```

> - The `Total Runtime` columns contains the total time in seconds taken to process the entire dataset at a specified concurrency level.

Reviewer: ```suggestion
- The `Total Runtime` columns contains the total time, in seconds, taken to process the entire dataset at a specified concurrency level.
```
